Topics/Development_Process/WSL.md

```diff
@@ -0,0 +1,113 @@
+# Learning WSL 2 Usage
+
+## Table of Contents
+### [Introduction](#introduction-1)
+### [Installation](#installation-1)
+### [Useful Commands](#useful-commands-1)
+### [WSL 1 Support](#wsl-1-support-1)
+### [Terminology](#terminology-1)
+----
+
+## Introduction
+
+**NOTE:** We assume your computer is already virtualization ready with technologies like Hyper-V Threading enabled. If you have a computer which may not obviously satisfy this (remote server, old systems, custom installations), we recommend 1) Enabling virtualization from BIOS/UEFI and then 2) Turning on Hyper-V Windows feature and Windows Subsystem for Linux feature. A simple search with the mentioned keywords would sufficiently provide the exact steps.
```

Comment:
it would be nice to leave a web link to where the user can access this information

Reply:
I will add some links !